src/molgenis/capice_resources/train_data_creator/__main__.py

```diff
@@ -107,10 +115,26 @@ def run_module(self, arguments):
             DatasetIdentifierEnums.VALIDATION.value: validation
         }
 
-    def export(self, output):
-        fake_vcf_header = files(
+    def create_fake_vcf_header(self):
+        """
+        Method to create the fake VCF header and add train-data-creator specific metadata.
+        Also sets the correct VCF date.
+        Returns:
+            header
+                Returns a single string of the VCF header, each metadata point separated by \\n with
+                the final metadata point ending in \\n.
+        """
+        header = files(
             'molgenis.capice_resources.train_data_creator.resources'
-        ).joinpath('fake_vcf_header.txt').read_text()
+        ).joinpath('fake_vcf_header.txt').read_text().strip().split('\n')
+        header.append(f'##CAPICE-Resources_version={__version__}')
+        header.append(f'##Used_ClinVar_File={self.input_clinvar_filename}')
+        header.append(f'##Used_VKGL_File={self.input_vkgl_filename}')
+        header.append(f'##fileDate={datetime.now().strftime("%Y%m%d")}\n')
```

Comment:
* While not strictly required, consider moving these just below the VCF-version line such as the example [here](https://samtools.github.io/hts-specs/VCFv4.2.pdf) and as done by ClinVar:
```
$ zcat clinvar.vcf.gz | head -n 4
##fileformat=VCFv4.1
##fileDate=2023-04-30
##source=ClinVar
##reference=GRCh37
```

* Perhaps we could add the reference build as well (either generic or simply stating which one from sources were used)?